pulp_ansible/app/downloaders.py

```diff
@@ -83,18 +83,10 @@ async def _run(self, extra_data=None):
         This method provides the same return object type and documented in
         :meth:`~pulpcore.plugin.download.BaseDownloader._run`.
 
-        Ansible token reference:
-            https://github.com/ansible/ansible/blob/devel/lib/ansible/galaxy/token.py
-
         """
-        if not self.ansible_token:
+        if not self.token:
```

Comment:
I believe doing 
```python
if not self.ansible_auth_url:
```
would cover @newswangerd comment.
We need to file an issue for this